DNN Platform/Website/admin/Skins/Logo.ascx.cs

```diff
@@ -22,16 +24,20 @@ namespace DotNetNuke.UI.Skins.Controls
     public partial class Logo : SkinObjectBase
     {
         private readonly INavigationManager _navigationManager;
-
+        private readonly string _cacheKey;
+        
         public Logo()
         {
             this._navigationManager = Globals.DependencyProvider.GetRequiredService<INavigationManager>();
+            this._cacheKey = string.Format(DataCache.PortalCacheKey, this.PortalSettings.PortalId, this.PortalSettings.CultureCode) + "LogoSvg";
         }
 
         public string BorderWidth { get; set; }
 
         public string CssClass { get; set; }
 
+        public Nullable<bool> isSVG { get; set; }
```

Comment:
I was thinking we should provide a property called injectSVG, if null or false we use the old behaviour, if true, we check the file extension and decide privatelly if it is an svg or not.

This would make it a non breaking change we can merge sooner and the way it is now, the theme developer would need to support only svg or not svg logos... which is not much reusable as it stands.

Reply:
Just a note here that public properties should be Title Cased so InjectSvg or IsSvg (https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)

My only concern with this property is that setting "IsSVG" in the skin object doesn't force the UI user to select an SVG image, so you could get unexpected results.

I think that InjectSvg as a property that then if an SVG image is provided it is injected would be the "least breaking" way to do this.

Reply:
I renamed the property to InjectSVG, but it does exactly what @valadas says.